src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayCompilerInternalOptions.cs

```diff
@@ -54,5 +54,12 @@ internal enum SymbolDisplayCompilerInternalOptions
         /// Append '!' to non-nullable reference types.
         /// </summary>
         IncludeNonNullableTypeModifier = 1 << 6,
+
+        /// <summary>
+        /// Don't append '!' (or '?') to non-nullable (or nullable) reference types.
+        /// Cannot be used in conjunction with <see cref="IncludeNonNullableTypeModifier"/>.
+        /// PROTOTYPE(NullableReferenceTypes): review design for this option before shipping. See https://github.com/dotnet/roslyn/issues/26198
+        /// </summary>
+        PlainReferenceTypes = 1 << 7,
```

Comment:
this seems odd given the option right above this.  Why not just have IncludeNonNullableTypeModifier and IncludeNullableTypeModifier?